plugins/action/include/plugins/action/action.h

```diff
@@ -106,6 +106,19 @@ class Action : public PluginBase {
      */
     ActionResult return_to_launch() const;
 
+    /**
+     * @brief Send command to reposition the vehicle to a specific WGS84 global position
+     *
+     * This sends the vehicle to a specified lattitude/longitude/altitude coordinates.
+     * @param latitude_deg Latitude in degrees
+     * @param longitude_deg Longitude in degrees
+     * @param altitude_amsl_m Altitude AMSL in meters
```

Comment:
Are you sure that this command is in absolute altitude and not relative? Could it be that we need to specify the frame when sending the command int?

Reply:
Also, I think [MAV_FRAME_GLOBAL_RELATIVE_ALT_INT](https://mavlink.io/en/messages/common.html#MAV_FRAME_GLOBAL_RELATIVE_ALT_INT) should be used in (https://github.com/Dronecode/DronecodeSDK/blob/27027d4eae32267616d6016cfebdcd13b0703719/core/mavlink_commands.h#L35)

Reply:
My guess is yes. It needs to be fixed on the PX4 side. I just have not figured out yet where it is exactly consumed.

Reply:
You're right, I filed an issue: https://github.com/PX4/Firmware/issues/10246.

Reply:
@mzahana oh yes that should be `_INT`, thanks for the note!